packages/lesswrong/components/comments/CommentsNode.tsx

```diff
@@ -182,7 +184,7 @@ const CommentsNode = ({
     if (forceSingleLine) return true;
     if (forceNotSingleLine) return false
 
-    return isTruncated && !isNewComment;
+    return isTruncated && !(expandNewComments && isNewComment);
```

Comment:
Stop new comments from taking up loads of space

Reply:
I'm still seeing some expanded comments (this is a local server with the prod DB) - is this intentional:
<img width="758" alt="Screenshot 2022-11-28 at 14 45 22" src="https://user-images.githubusercontent.com/5075628/204306694-97ec45a5-2ea8-43a6-bc81-cc0dbf4c9f89.png">
<img width="745" alt="Screenshot 2022-11-28 at 14 45 31" src="https://user-images.githubusercontent.com/5075628/204306707-5dee58b6-f316-4377-a3a4-4bd2e76c78ed.png">

Reply:
This is intentional, the only thing I changed was to stop all "unread" comments from being expanded, some of them still are. (I think the default behaviour is that only the most recent comment is expanded, not sure though)